peekingduck/pipeline/nodes/draw/bbox.py

```diff
@@ -17,19 +17,23 @@
 
 from typing import Any, Dict
 from peekingduck.pipeline.nodes.node import AbstractNode
-from peekingduck.pipeline.nodes.draw.utils.drawfunctions import draw_bboxes
+from peekingduck.pipeline.nodes.draw.utils.bbox import draw_bboxes
+from peekingduck.pipeline.nodes.draw.utils.constants import CHAMPAGNE
 
 
 class Node(AbstractNode):
     """Draw node for drawing bboxes onto image"""
 
     def __init__(self, config: Dict[str, Any]) -> None:
         super().__init__(config, node_path=__name__)
-        self.bbox_thickness = config["bbox_thickness"]
-        self.bbox_color = tuple(config["bbox_color"])
+        self.show_labels = config['show_labels']
+        if self.show_labels:
+            self.inputs.append("bbox_labels")
```

Comment:
I think it would be more consistent with our docs to remove this part and include "bbox_labels" in config? It seems somewhat unusual to add it here - our users are going to wonder how it has access to bbox labels if it's not in config.

When using pose models with draw.bbox, it will throw an error because pose models don't return bbox_labels - perhaps we should alter posenet to return bbox_labels and it is fixed at "person"?

Let me know what you think.

Reply:
I initially did this too and got the same error, therefore switched to a simple toggle. 
If we do add labels to pose models, would it affect the inference speed? Also this would need to be done to all pose estimation models.
I feel that when using pose, you don't really wanna have the label, since its implicit and clogs up the screen from multi pose.
For cases of using labels, it is really more of when we do object detection (multiple objects), therefore I chose to confine it to toggling for the relevant situations and setting default to false, so that it works even when using pose without the labels.

Reply:
As per discussion, using append to add additional input requirements goes against the direction of using configs to manage it. It will be confusing for users as well. As such, changed to use bbox_label in posenet, and made changes accordingly